config/crd/bases/cloudsql.cloud.google.com_authproxyworkloads.yaml

```diff
@@ -905,6 +905,13 @@ spec:
                     telemetry:
                       description: Telemetry specifies how the proxy should expose telemetry. Optional, by default
                       properties:
+                        adminPort:
+                          description: AdminPort the port for the /quitquitquit endpoint. This sets the proxy container's CLI argument `--admin-port`
```

Comment:
Let's match the Proxy description for this flag. Also no quitquitquit here.

Reply:
Fixed.